Example/Tests/SnapshotTests.swift

```diff
@@ -32,7 +32,7 @@ class SnapshotTests: FBSnapshotTestCase {
     }
     
     func testEmptySampleTableViewController() {
-        UIApplication.shared.keyWindow?.rootViewController = sampleTableViewController
+        UIApplication.shared.windows.filter {$0.isKeyWindow}.first?.rootViewController = sampleTableViewController
```

Comment:
I think `first(where:)` would be a bit more performant, since it returns immediately after found the first element that matches the predicate, unlike filter, that needs to continue looking all the views, to later get the first element.